cql3/functions/functions.cc

```diff
@@ -117,15 +117,15 @@ functions::init() noexcept {
     return ret;
 }
 
-void functions_changer::add_function(shared_ptr<function> func) {
+void functions::add_function(shared_ptr<function> func) {
     if (find(func->name(), func->arg_types())) {
         throw std::logic_error(format("duplicated function {}", func));
     }
     _declared.emplace(func->name(), func);
 }
 
 template <typename F>
-void functions_changer::with_udf_iter(const function_name& name, const std::vector<data_type>& arg_types, F&& f) {
+void functions::with_udf_iter(const function_name& name, const std::vector<data_type>& arg_types, F&& f) {
```

Comment:
Unrelated, preexisting: it's better to use std::function rather than a template, to prevent code bloat.

Reply:
refactored